src/Markup/Perspex.Markup.Xaml/Converters/TimeSpanTypeConverter.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) The Perspex Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using OmniXaml.TypeConversion;
+using Perspex.Media;
+
+namespace Perspex.Markup.Xaml.Converters
+{
+    public class TimeSpanConverter : ITypeConverter
+    {
+        public bool CanConvertFrom(IXamlTypeConverterContext context, Type sourceType)
+        {
+            return sourceType == typeof(string);
+        }
+
+        public bool CanConvertTo(IXamlTypeConverterContext context, Type destinationType)
+        {
+            return false;
+        }
+
+        public object ConvertFrom(IXamlTypeConverterContext context, CultureInfo culture, object value)
+        {
+            // TimeSpan does not parse seconds format directly, restrict
+            // syntax to seconds for now (ie. "0.25")
+            var secs = double.Parse((string)value);
```

Comment:
We could. However in other animation libraries I have seen it is generally accepted to use seconds, as you likely never want to have an animation more than a minute long in general. Seconds format is shorter. We could also consider changing the Duration property to double and commenting that it is in seconds - then we would not need the type converter at all.